package.json

```diff
@@ -49,7 +49,7 @@
     "node": ">=6"
   },
   "scripts": {
-    "install": "node-pre-gyp install --fallback-to-build=false || make node",
+    "install": "node-pre-gyp install --fallback-to-build=false",
```

Comment:
This change means that `npm install` will still work and fetch a binary. But if a binary is not found the install will exit instead of calling `make node`.